Gems/ROS2/Code/Include/ROS2/Lidar/LidarRegistrarBus.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/string/string.h>
+
+namespace ROS2
+{
+    //! Structure used to describe LidarSystem's feature support.
+    struct LidarSystemFeatures
+    {
+    public:
```

Comment:
nit: `public` can be omitted as this is a struct and public be default

Reply:
Thanks! Fixed.